Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -778,6 +886,9 @@ def validate_fix_severity_value(severity: str, indicator_value: str | None = Non
 
 def main():  # pragma: no cover
     params = demisto.params()
+    if params.get('feed') and params.get('feedFetchInterval') and arg_to_number(params.get('feedFetchInterval')) < 20:
+        demisto.debug("If feedFetchInterval parameter is set to less then 15 minutes, it could lead to internal error from "
```

Comment:
some corrections:
1. change to info level
2. print he current interval in the log and state that it is lower than 20 and the implications.
3. call the parameter by its display value. not "feedFetchInterval", but "Feed Fetch Interval"

Reply:
feed_fetch_interval = arg_to_number(params.get('feedFetchInterval'))
    if params.get('feed') and feed_fetch_interval and arg_to_number(feed_fetch_interval) < 20:
        demisto.info(f"`Feed Fetch Interval` is set to {feed_fetch_interval} which is less then 15 minutes. This could lead to "
                     "internal error from xdr side.")